test/end-to-end/company.js

```diff
@@ -100,6 +100,32 @@ describe("Company application endpoint", () => {
                 fs.unlinkSync(filename);
             });
 
+            test("should finish the application with single contact", async () => {
+                await test_agent
+                    .post("/company/application/finish")
+                    .attach("logo", "test/data/logo-niaefeup.png")
+                    .field("bio", "A very interesting and compelling bio")
+                    .field("contacts", "contact1")
+                    .expect(HTTPStatus.OK);
+
+                const test_company = [... await Company.find({})][0];
+                expect([...test_company.contacts]).toEqual(["contact1"]);
+                expect(test_company.hasFinishedRegistration).toBe(true);
+                expect(test_company.bio).toBe("A very interesting and compelling bio");
+                const filename = path.join(__dirname, `../../${test_company.logo}`);
+                expect(fs.existsSync(filename)).toBe(true);
+
+                await test_agent
+                    .post("/company/application/finish")
+                    .attach("logo", "test/data/logo-niaefeup.png")
+                    .field("bio", "A very interesting and compelling bio")
+                    .field("contacts", ["contact1", "contact2"])
+                    .expect(HTTPStatus.FORBIDDEN);
```

Comment:
Please validate the error message (and in the previous test, I hadn't noticed), please. Other than that, it can be merged